### PR TITLE
Magento_Theme: avoid using deprecated escape* methods from AbstractBlock

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Theme/view/adminhtml/templates/browser/content/files.phtml [hunk 1]
```diff
@@ -1,24 +1,27 @@
-<?php
+\<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Theme/view/adminhtml/templates/browser/content/files.phtml]: Please revert this line
```suggestion
<?php
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Theme/view/adminhtml/templates/browser/content/files.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Theme/view/adminhtml/templates/browser/content/files.phtml]: reverted

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests EE

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
could you resolve conflict?

**Comment by sergeynezbritskiy** [comment]
> could you resolve conflict?

done

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Database Compare, Database Compare

**Comment by ihor-sviziev** [comment]
@magento run WebAPI Tests

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8659](https://jira.corp.magento.com/browse/ENGCOM-8659) has been created to process this Pull Request

**Comment by ihor-sviziev** [comment]
Database Compare

**Comment by ihor-sviziev** [comment]
@magento run Database Compare

**Pull request closed without merging**